source/docs/software/advanced-controls/system-identification/loading-data.rst

```diff
@@ -0,0 +1,26 @@
+Loading Data
+============
+
+After downloading the WPILog containing the tests from the roboRIO, go to the ``Log Loader`` pane in SysId and click ``Open data log file...``.
+
+.. note:: If the log file has many entries, use the ``Filter`` box to search for the needed entries.
```

Comment:
Lets move this note to after we instruct the user on selecting the data

Reply:
It could be needed for finding the data to select, in which case it is most needed right after the log is loaded into the tool